SxGDriveTokenGenerator.py

```diff
@@ -30,14 +25,12 @@ class Config:
         client_secret_paths,
         output_folder_path,
         scopes,
-        output_token_json_name,
-        output_client_secret_json_name"""

```

Comment:
Removed as it is not needed. The `gdrive.token` now stores the whole credential JSON.